spec/generators/install_generator_spec.rb

```diff
@@ -18,35 +20,59 @@ module Admin
           class ApplicationController < Administrate::ApplicationController
       RB
     end
+
+    it "uses the namespace option" do
+      stub_generator_dependencies
+      controller = file("app/controllers/manager/application_controller.rb")
+
+      run_generator ["--namespace", "manager"]
+
+      expect(controller).to exist
```

Comment:
I like this verification as it will raise a proper error message when the file doesn't exist (`expected "/home/rodrigo/Projects/administrate/tmp/app/controllers/manager/application_controller.rb" to exist`)

If we use only the `have_correct_syntax` matcher and the generator fails to create the file, the error message would be `No such file or directory @ rb_sysopen - /home/rodrigo/Projects/administrate/tmp/app/controllers/manager/application_controller.rb`

What do you think?